test/e2e/apps/daemon_restart.go

```diff
@@ -60,53 +60,68 @@ const (
 // restartDaemonConfig is a config to restart a running daemon on a node, and wait till
 // it comes back up. It uses ssh to send a SIGTERM to the daemon.
 type restartDaemonConfig struct {
-	nodeName     string
-	daemonName   string
-	healthzPort  int
-	pollInterval time.Duration
-	pollTimeout  time.Duration
+	nodeName            string
+	daemonName          string
+	insecureHealthzPort int
+	secureHealthzPort   int
+	pollInterval        time.Duration
+	pollTimeout         time.Duration
 }
 
 // NewRestartConfig creates a restartDaemonConfig for the given node and daemon.
-func NewRestartConfig(nodeName, daemonName string, healthzPort int, pollInterval, pollTimeout time.Duration) *restartDaemonConfig {
+func NewRestartConfig(nodeName, daemonName string, insecureHealthzPort int, secureHealthzPort int, pollInterval, pollTimeout time.Duration) *restartDaemonConfig {
```

Comment:
would prefer to use `insecureHealthzPort, secureHealthzPort int` to be consistent